etl_manager/meta.py

```diff
@@ -447,6 +492,9 @@ def to_dict(self):
             "columns": self.columns,
             "location": self.location,
         }
+        if bool(self.sensitivity):
+            meta["sensitivity"] = self.sensitivity
```

Comment:
Again if agree with suggestion on no setter for sensitivity then I'd imagine this bit would call `self.update_sensitivity` then check if `self.sensitivity == meta["sensitivity"]`

Reply:
I think the existing code is the correct way to output table sensitivity in a dict, though I'm not sure we'd want to do that, given you couldn't then use the same dict as an input to create a new `TableMeta` object.